### PR TITLE
Zap UI improvements + cleanup

### PR DESCRIPTION
**Pull request by Padraic-O-Mhuiris**

### CODE REVIEW

**Comment by sentilesdal** [comment]
#### 📝 Changed routes:


| [**…principalTokenAddress\]**](https://elf-frontend-staging-m0es1t4cw-element-finance.vercel.app/fixedrates/[principalTokenAddress]) |
|:-:|
|<a href="https://elf-frontend-staging-m0es1t4cw-element-finance.vercel.app/fixedrates/[principalTokenAddress]">Screenshot of /fixedrates/[principalTokenAddress]</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/N55ko5ZoDw4l0ZPm8fxGNXKJM3Zgwlfayr-qhWDfA9Muf-sDVpXRGnxEvxgZPxeHZd5NjL_PVX5xiRTklvOCzg.png">(view full size)</a> |







Commit 6ff89c02f76bbbb8135a3422acc71bc27d9bf506 (https://elf-frontend-staging-m0es1t4cw-element-finance.vercel.app).

File: src/ui/fixedrates/grid.module.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .fixedRatesGrid {
   display: grid;
   /* principal token, term, fixed apr, action button */
-  grid-template-columns: 320px 128px 128px auto;
+  grid-template-columns: 180px 120px 128px 100px 10px;
```
Comment (DannyDelott) [inline src/ui/fixedrates/grid.module.css:12]: minor nit: update the comment above to include the new column name

File: src/ui/token/TokenIcon.tsx [hunk 2]
```diff
@@ -29,6 +34,96 @@ export interface IconProps {
...
 
+export function StethIcon({
+  height,
+  width,
+  className,
+  style,
+}: IconProps): ReactElement {
+  return (
+    <SvgIcon
+      alt={"mim"}
```
Comment (DannyDelott) [inline src/ui/token/TokenIcon.tsx]: "steth"

**Review by DannyDelott — APPROVED** [review]
a few small nits, otherwise looks good!

**Pull request merged**